docs/rules/brace-style.md

```diff
@@ -40,27 +40,27 @@ While no style is considered better than the other, most developers agree that h
 
 ## Rule Details
 
-This rule is aimed at enforcing a particular brace style in JavaScript. As such, it warns whenever it sees a statement or declaration that does not adhere to the one true brace style.
+This rule is aimed at enforcing a particular brace style in JavaScript.
 
 ## Options
 
-The rule takes two options:
+This rule has a string option:
 
-1. A string which must be either `"1tbs"`, `"stroustrup"` or `"allman"`. The default is `"1tbs"`.
-2. An object that further controls the behaviour of this rule. Currently, the only available parameter is `allowSingleLine`, which indicates whether start and end braces may be on the same line.
+* `"1tbs"` (default) enforces one true brace style
+* `"stroustrup"` enforces Stroustrup style
+* `"allman"` enforces Allman style
 
-You can set the style in configuration like this:
+This rule can have an object option for an exception:
```

Comment:
Yes, “can have” was a feeble attempt to imply the additional option is optional (groan ;-)

Readers will see that from the eslint comments in the example code, true?